modules/creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-with-aws-kms-key"]
+= Creating a ROSA cluster using a custom AWS KMS key
+
+You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that is used to encrypt either node root volumes, the etcd database, or both. A different KMS key ARN can be provided for each option.
+
+[NOTE]
+====
+{hcp-title} does not automatically configure the `default` StorageClass to encrypt persistent volumes with the customer-provided KMS key. This is something that can be configured in-cluster after installation.
+====
+
+
+.Procedure
+
+. Create a custom AWS customer-managed KMS key by running the following command:
++
+[source,terminal]
+----
+$ KMS_ARN=$(aws kms create-key --region $AWS_REGION --description 'Custom ROSA Encryption Key' --tags TagKey=red-hat,TagValue=true --query KeyMetadata.Arn --output text)
+----
++
+This command saves the Amazon Resource Name (ARN) output of this custom key for further steps.
++
+[NOTE]
+====
+Customers provide the `--tags TagKey=red-hat,TagValue=true` that are required for a customer KMS key.
```

Comment:
It seems like you're missing a noun after `--tags TagKey=red-hat,TagValue=true`. Should it be "arguments"?

Reply:
Probably should be

```suggestion
Customers must provide the `--tags TagKey=red-hat,TagValue=true` argument that are required for a customer KMS key.
```

Reply:
Applied:
Customers must provide the `--tags TagKey=red-hat,TagValue=true` argument that is required for a customer KMS key.